frontend/src/slices/orders/actions.ts

```diff
@@ -0,0 +1,44 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+import { getErrorMessage } from '~/libs/helpers/helpers.js';
+import { notification } from '~/libs/packages/notification/notification.js';
+import { type AsyncThunkConfig } from '~/libs/types/types.js';
+import {
+  type OrderCalculatePriceRequestDto,
+  type OrderCalculatePriceResponseDto,
+  type OrderCreateRequestDto,
+  type OrderResponseDto,
+} from '~/packages/orders/orders.js';
+
+import { name as sliceName } from './order.slice.js';
+
+const createOrder = createAsyncThunk<
+  OrderResponseDto,
+  OrderCreateRequestDto,
+  AsyncThunkConfig
+>(`${sliceName}/create-order`, async (payload, { extra }) => {
+  const { ordersApi } = extra;
+
+  try {
+    const result = await ordersApi.createOrder(payload);
+
+    notification.success('Order successfully created');
+
+    return result;
+  } catch (error) {
+    notification.warning(getErrorMessage(error));
```

Comment:
Perhaps it is better to use  notification.error ?
```suggestion
    notification.error(getErrorMessage(error));
```